precariobelasting/precariobelasting.json

```diff
@@ -0,0 +1,193 @@
+{
+  "type": "dataset",
+  "id": "precariobelasting",
+  "title": "precariobelasting",
+  "status": "beschikbaar",
+  "description": "precariobelasting",
+  "version": "0.0.1",
+  "crs": "EPSG:28992",
+  "tables": [
+    {
+      "id": "terrassen",
+      "type": "table",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["schema", "id"],
+        "display": "id",
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "id": {
+            "type": "integer",
+            "description": "technische identificatie"
+          },
+          "geometry": {
+            "$ref": "https://geojson.org/schema/Geometry.json",
+            "description": "polygoon percariobelastinggebied"
```

Comment:
Deze description is ook meer technisch en voegt weinig semantisch begrip toe. Wat mij betreft mag je het weglaten dan. 

Description is wat je zou willen tonen in een tooltip/info icoontje met extra informatie over een specifiek veld... en dit zou daarin weinig toevoegen.

Reply:
Eens, verwijderd